packages/textcomplete-core/src/Dropdown.ts

```diff
@@ -316,15 +316,15 @@ class DropdownItem {
     span.innerHTML = this.searchResult.render()
     li.appendChild(span)
 
-    li.addEventListener("click", this.onClick)
+    li.addEventListener("mousedown", this.onClick)
 
     this.el = li
   }
 
   destroy(): this {
     const li = this.el
     li.parentNode?.removeChild(li)
-    li.removeEventListener("click", this.onClick, false)
+    li.removeEventListener("mousedown", this.onClick, false)
     return this
```

Comment:
regression from 0.1.12 that i switched back.

https://github.com/yuku/textcomplete/commit/8d970fca508557e610a6a3468854a6e784dbada8